chainio/clients/elcontracts/reader_test.go

```diff
@@ -114,6 +116,314 @@ func TestChainReader(t *testing.T) {
 		assert.NoError(t, err)
 		assert.NotEmpty(t, digest)
 	})
+
+	t.Run("get staker shares", func(t *testing.T) {
+		strategies, shares, err := clients.ElChainReader.GetStakerShares(
+			ctx,
+			common.HexToAddress(operator.Address),
+		)
+		assert.NotZero(t, len(strategies))            // Strategies has at least one element
```

Comment:
You can add a message to the assertion, instead of the comment.
```suggestion
		assert.NotZero(t, len(strategies), "Strategies has at least one element")
```

Reply:
Done in [Turn comments into assert messages](https://github.com/Layr-Labs/eigensdk-go/pull/429/commits/64b4c622b4405c601bb99af1c7aa11d184b742b8)